apps/web/components/DefaultHeader/DefaultHeader.tsx

```diff
@@ -32,6 +32,7 @@ export interface DefaultHeaderProps {
   imageObjectPosition?: 'left' | 'center' | 'right'
   className?: string
   titleClassName?: string
+  logoImageClassName?: boolean
```

Comment:
Can this be a string instead and we'll use vanilla extract to apply the styles?

Doesn't make a lot of sense to to have a "logoImageClassName" boolean